src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/processes-procedures/tech-writer-workflow.mdx

```diff
@@ -0,0 +1,134 @@
+
+## Run the site locally
+
+You will use the Terminal to build the site locally. This will allow you to preview changes before opening a PR. While running, the site will reflect any local changes made. Node and Yarn are tools we use to build the site on your local machine.
+
+### Prerequisites
+- [Install Node](https://nodejs.org/en/download/)
+- Install Yarn 
+  - `npm install -g yarn`
+
+### Build the site
+1. In your Terminal, change your directory `cd` to your cloned repo, **docs-website**. 
+    - `cd ~/Documents/GitHub/docs-website`
+2. Run yarn with the following commands:
+    - `yarn`
+    - `yarn start`
+3. The site will take a few minutes to build. Once it finishes building, you can access your preview site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
+
+## Work on a branch, not a fork
+Some teams work on branches, some teams work on forks; the docs team works in branches. As long as a branch has been pushed upstream, this allows us to work collaboratively and ensure that no work is ever lost when someone goes on FTO.
+
+To create a branch on the docs-website repo:
+1. Open Github Desktop
+2. Click on **Current branch: xxx**
+3. Click on **New Branch**
+4. You will be prompted to name your new branch. Ensure the name is relevant to the work done. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
+
+## Edit a doc
+
+Once your local environment and branch are set up, you are ready to edit a doc. Check out the docs site style guide for writing guidelines.
+
+1. First, ensure your **Current Branch** in Github Desktop is set to the correct branch, not **Develop**.
+2. Navigate to the doc you want to edit in Finder. If I wanted to edit a Python agent doc, I would navigate to: `~/Documents/Github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
+3. Edit the doc in your text-editor of choice. You should write docs in markdown language. Reference the style guide for help with formatting markdown
+4. Save the file with your edits, then follow the same process for any other docs you wish to edit.
+
+## Committing your changes
+
+Once your edits are done, you can commit them. This stages your changes, which you will later push upstream.
+
+1. Navigate to Github Desktop. The left column should have a record of all the edits you have made to docs. 
+2. In the bottom left corner, name your commit and add a good description of your edits. It should be descriptive enough to ensure that someone can understand all the changes made by simply scanning this description.
+3. Press **Commit to yourbranchname**
+
+## Publish your commits
+
+Once you have committed your changes, you are almost ready to open your Pull Request.
```

Comment:
Something about how a Pull Request will get your changes to our Docs team. I just feel like it's good to ground this stuff in the Docs publishing process.